cpp/oneapi/dal/policy.hpp

```diff
@@ -39,25 +40,47 @@ enum class cpu_extension : uint64_t {
     avx512 = 1U << 5
 };
 
-class ONEAPI_DAL_EXPORT default_execution_context : public base {
+class ONEAPI_DAL_EXPORT default_policy : public base {
```

Comment:
Why do we need to have `default_policy` and `data_parallel_policy` in public API? I thought we discussed it and agreed that if user wants running with DPC++ he sends a queue to the `train()`, and sends nothing when wants to run on CPU. It's inconsistent with data management API

Reply:
I'm afraid we'll need to have it in public API...

The reason is that policy for CPU is used to control at least threading (number of threads) and available instruction sets.